dependencies.rst

```diff
@@ -65,6 +65,10 @@ Packages Not On PyPI
 
 Sometimes you'll want to use packages that are properly arranged with setuptools, but aren't published to PyPI. In those cases, you can specify a list of one or more ``dependency_links`` URLs where the package can be downloaded, along with some additional hints, and setuptools will find and install the package correctly.
 
+*Edit on 2017-03-23 : Dependency Links are now considered as depecrated in Python Wheels*
```

Comment:
I corrected the typo in new patch. 
Also, added a text mentioning "pip install" would no longer consider dependency_links during installation.